src/main/java/com/osiris/dyml/YamlSection.java

```diff
@@ -166,41 +166,52 @@ public YamlSection addKeys(String... keys) {
         return this;
     }
 
+    /**
+     * See {@link #addValues(List)} for details.
+     */
+    public <T> YamlSection addValues(T... v) {
+        if (v != null)
+            return addValues(utils.arrayToValuesList(v));
```

Comment:
should call addSValues(v)
otherwise infinite loop.
I also saw this same issue in the other functions below.